src/main.ts

```diff
@@ -2,12 +2,18 @@ import { ValidationPipe } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import * as cookieParser from 'cookie-parser';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule, { cors: true });
+  app.enableCors({
+    credentials: true,
+    origin: '*',
```

Comment:
What's the reasoning behind this?

Reply:
My mistake; I added the wildcard flag for testing requests from the browser, this should have origins set explicitly; I'll change this to an environment variable, with a URL to the accepted origin